### PR TITLE
Improve Membership columns with long words or text

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Fix bug #1388340 
Improves word wrapping within container columns with or w/o hyphenation or natural breaks.

long user names:
screen shot 2016-11-01 at 12 08 27 pm 1

long service account names (namespace / name): 
screen shot 2016-11-01 at 12 15 55 pm

@jwforres @spadgett

### CODE REVIEW

File: app/styles/_membership.less [hunk 1]
```diff
@@ -31,7 +31,7 @@
     .col-name {
       min-width: 165px;
       padding: 0 5px 10px 0;
-      word-wrap: break-word;
+      word-break: break-all;
```
Comment (jwforres) [inline app/styles/_membership.less]: is there a reason you aren't using our word-break (i think is the name) mixin? it makes sure the best option is used for each browser

Comment (spadgett) [inline app/styles/_membership.less]: Yeah, there are the `word-break` and `word-break-all` mixins. I'd prefer to use the former unless we really need `break-all`.

Comment (benjaminapetersen) [inline app/styles/_membership.less]: @spadgett Yeah, `word-wrap: break-word;` doesn't get the job done with the non-hyphenated words.
@jwforres Mmm... I just forgot about it :)

Let me look @ the mixin quick and see if that does the job.

Comment (benjaminapetersen) [inline app/styles/_membership.less]: With `word-wrap: break-word` you get this:
screen shot 2016-11-01 at 1 16 43 pm

Comment (benjaminapetersen) [inline app/styles/_membership.less]: Updated w/mixin.

File: app/styles/_membership.less [hunk 2]
```diff
@@ -100,7 +99,7 @@
         }
       }
       .col-name {
-        min-width: 180px;
+        width: 180px;
```
Comment (spadgett) [inline app/styles/_membership.less]: Per our conversation, let's make the name width larger.

**Comment by benjaminapetersen** [comment]
@sg00dwin happy to take any of your suggestions as well!

File: app/styles/_membership.less (hunk 2 above)
Comment (benjaminapetersen) [inline app/styles/_membership.less]: Also removing the max width on the content area.

**Comment by benjaminapetersen** [comment]
@spadgett @sg00dwin here's where its at at this point:

50% for names in readonly mode:
screen shot 2016-11-01 at 3 21 12 pm
additional width in edit mode:
screen shot 2016-11-01 at 3 21 19 pm

tablet/mobile the same as before.

**Comment by benjaminapetersen** [comment]
Additionally, this PR fixes an issue with column collapse in mobile IE:

before:
screen shot 2016-11-03 at 4 02 35 pm
after:
screen shot 2016-11-03 at 4 02 59 pm

**Comment by benjaminapetersen** [comment]
rebased

File: app/styles/_membership.less [hunk 3]
```diff
@@ -79,10 +80,12 @@
       }
     }
   }
+  .half-width {
+    min-width: 50% !important;
```
Comment (spadgett) [inline app/styles/_membership.less]: Do we really need `!important`? Does setting `flex-basis` instead work?

Comment (benjaminapetersen) [inline app/styles/_membership.less]: Its a class conditionally set:
```html
ng-class="{ 'half-width': !mode.edit }"
```
Setting `flex-basis: 50%` doesn't have the same effect.  There may be a better way to do it, but it is descriptive of my intent.  

Thinking on it...

Comment (benjaminapetersen) [inline app/styles/_membership.less]: Updating to this:
```
.col-name {
      &.half-width {
        min-width: 50%;
      }
}
```

Comment (benjaminapetersen) [inline app/styles/_membership.less]: that works, done

**Comment by spadgett** [comment]
[merge]

**Pull request merged**